### PR TITLE
HBASE-23645 Fixed remaining Checkstyle violations in hbase-common tests

### PR DESCRIPTION
**Pull request by HorizonNet**
Backport of #989 to branch-1.

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 24s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 4 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 26s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   7m 54s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   0m 28s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  compile  |   0m 34s |  branch-1 passed with JDK v1.7.0_252  |
| +1 :green_heart: |  checkstyle  |   5m 58s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   3m 16s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 34s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 35s |  branch-1 passed with JDK v1.7.0_252  |
| +0 :ok: |  spotbugs  |   1m  5s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +0 :ok: |  findbugs  |   0m 24s |  branch/hbase-checkstyle no findbugs output file (findbugsXml.xml)  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m  9s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 28s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javac  |   0m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 35s |  the patch passed with JDK v1.7.0_252  |
| +1 :green_heart: |  javac  |   0m 35s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   5m 55s |  root: The patch generated 0 new + 1 unchanged - 29 fixed = 1 total (was 30)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   3m  6s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   5m 21s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 29s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 35s |  the patch passed with JDK v1.7.0_252  |
| +0 :ok: |  findbugs  |   0m 12s |  hbase-checkstyle has no data from findbugs  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 11s |  hbase-checkstyle in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 34s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 23s |  The patch does not generate ASF License warnings.  |
|  |   |  51m  0s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1075 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux da78c439ca9e 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1075/out/precommit/personality/provided.sh |
| git revision | branch-1 / 99a59cf |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/1/artifact/out/xml.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/1/testReport/ |
| Max. process+thread count | 164 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-common U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/1/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/test/java/org/apache/hadoop/hbase/TestTimeout.java [hunk 1]
```diff
@@ -18,31 +18,33 @@
...
-    public void infiniteLoop() {
-        while (true) {}
-   }
+  /**
+   * Enable to check if timeout works.
+   * Can't enable as it waits 30seconds and expected doesn't do Exception catching
+   */
+  @Ignore
+  @Test
+  public void infiniteLoop() {
```
Comment (saintstack) [inline hbase-common/src/test/java/org/apache/hadoop/hbase/TestTimeout.java]: I remember this one.....

File: hbase-common/src/test/java/org/apache/hadoop/hbase/Waiter.java [hunk 2]
```diff
@@ -87,29 +85,25 @@ public static float getWaitForRatio(Configuration conf) {
    */
   @InterfaceAudience.Private
   public interface Predicate<E extends Exception> {
-
     /**
      * Perform a predicate evaluation.
      * @return the boolean result of the evaluation.
-     * @throws Exception thrown if the predicate evaluation could not evaluate.
+     * @throws E thrown if the predicate evaluation could not evaluate.
```
Comment (saintstack) [inline hbase-common/src/test/java/org/apache/hadoop/hbase/Waiter.java]: Good

**Comment by HorizonNet** [comment]
@saintstack It is related to [HBASE-22732](https://issues.apache.org/jira/browse/HBASE-22732). We get it on every PR targeting branch-1, which changes the suppressions file.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |  21m 45s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 4 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 30s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   8m 11s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   0m 33s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  compile  |   0m 40s |  branch-1 passed with JDK v1.7.0_252  |
| +1 :green_heart: |  checkstyle  |   6m 21s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   4m  9s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 39s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 39s |  branch-1 passed with JDK v1.7.0_252  |
| +0 :ok: |  spotbugs  |   1m 14s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +0 :ok: |  findbugs  |   0m 26s |  branch/hbase-checkstyle no findbugs output file (findbugsXml.xml)  |
| -0 :warning: |  patch  |   1m 30s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 35s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 34s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javac  |   0m 34s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 40s |  the patch passed with JDK v1.7.0_252  |
| +1 :green_heart: |  javac  |   0m 40s |  the patch passed  |
| -1 :x: |  checkstyle  |   6m 22s |  root: The patch generated 1 new + 0 unchanged - 30 fixed = 1 total (was 30)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   3m 49s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   6m 26s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 36s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 42s |  the patch passed with JDK v1.7.0_252  |
| +0 :ok: |  findbugs  |   0m 14s |  hbase-checkstyle has no data from findbugs  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 13s |  hbase-checkstyle in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 50s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 27s |  The patch does not generate ASF License warnings.  |
|  |   |  77m 58s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1075 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux 9bbbcc270f40 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1075/out/precommit/personality/provided.sh |
| git revision | branch-1 / 2e3273f |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/2/artifact/out/diff-checkstyle-root.txt |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/2/artifact/out/xml.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/2/testReport/ |
| Max. process+thread count | 171 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-common U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/2/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 27s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 4 new or modified test files.  |
||| _ branch-1 Compile Tests _ |
| +0 :ok: |  mvndep  |   1m 22s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   7m 43s |  branch-1 passed  |
| +1 :green_heart: |  compile  |   0m 31s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  compile  |   0m 35s |  branch-1 passed with JDK v1.7.0_252  |
| +1 :green_heart: |  checkstyle  |   6m 29s |  branch-1 passed  |
| +1 :green_heart: |  shadedjars  |   3m 43s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 33s |  branch-1 passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 39s |  branch-1 passed with JDK v1.7.0_252  |
| +0 :ok: |  spotbugs  |   1m 10s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +0 :ok: |  findbugs  |   0m 26s |  branch/hbase-checkstyle no findbugs output file (findbugsXml.xml)  |
| -0 :warning: |  patch  |   1m 23s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m  6s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 28s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javac  |   0m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 35s |  the patch passed with JDK v1.7.0_252  |
| +1 :green_heart: |  javac  |   0m 35s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   5m 50s |  root: The patch generated 0 new + 0 unchanged - 30 fixed = 0 total (was 30)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  xml  |   0m  0s |  The patch has 1 ill-formed XML file(s).  |
| +1 :green_heart: |  shadedjars  |   3m  8s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |   5m 13s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2.  |
| +1 :green_heart: |  javadoc  |   0m 27s |  the patch passed with JDK v1.8.0_242  |
| +1 :green_heart: |  javadoc  |   0m 34s |  the patch passed with JDK v1.7.0_252  |
| +0 :ok: |  findbugs  |   0m 12s |  hbase-checkstyle has no data from findbugs  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 13s |  hbase-checkstyle in the patch passed.  |
| +1 :green_heart: |  unit  |   2m 30s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 22s |  The patch does not generate ASF License warnings.  |
|  |   |  51m 38s |   |


| Reason | Tests |
|-------:|:------|
| XML | Parsing Error(s): |
|   | hbase-checkstyle/src/main/resources/hbase/checkstyle-suppressions.xml |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1075 |
| Optional Tests | dupname asflicense checkstyle javac javadoc unit xml spotbugs findbugs shadedjars hadoopcheck hbaseanti compile |
| uname | Linux 111011621c10 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1075/out/precommit/personality/provided.sh |
| git revision | branch-1 / ee50d48 |
| Default Java | 1.7.0_252 |
| Multi-JDK versions | /usr/lib/jvm/zulu-8-amd64:1.8.0_242 /usr/lib/jvm/zulu-7-amd64:1.7.0_252 |
| xml | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/3/artifact/out/xml.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/3/testReport/ |
| Max. process+thread count | 102 (vs. ulimit of 10000) |
| modules | C: hbase-checkstyle hbase-common U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1075/3/console |
| versions | git=1.9.1 maven=3.0.5 findbugs=3.0.1 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**